identity/src/main/java/com/android/identity/VerificationHelper.java

```diff
@@ -396,6 +412,11 @@ public void connect(@NonNull DataRetrievalAddress address) {
         } else if (mDataTransport instanceof DataTransportBle) {
             // Set the preference for using L2CAP
             ((DataTransportBle) mDataTransport).setUseL2CAPIfAvailable(mUseL2CAP);
+            ((DataTransportBle) mDataTransport).setClearCache(mBleClearCache);
+            if (mBleClearCache && mDataTransport instanceof DataTransportBleCentralClientMode) {
+                mLog.info("Ignoring bleClearCache flag since it only applies to "
```

Comment:
This is a duplicate log to what's in PresentationHelper, right?

Seems like we should only log it in one spot.

Reply:
It's actually not entirely the same message and for PresentationHelper it's ignored if using peripheral-server-mode, here it's ignored if using central-client-mode. So we can't move the logging message into those classes because they don't really know how they are used.

Reply:
Ah, ok.